localinfo/views.py

```diff
@@ -33,3 +33,17 @@ def get(self, request):
             faqs = {"faqs": search_faq(search),
                     "query": search}
         return render(request, template, faqs)
+
+
+class CustomRouteCsvUploader(View):
+
+    def post(self, request):
+        csv_file = request.FILES.get('csvDictionary', False)
+        if csv_file is not False:
```

Comment:
remplazar `if csv_file is not False` por `if csv_file:`

Reply:
dentro del if usar lo que pusiste en el otro archivo:
````
        f = open(csv_file.name, 'r')
        reader = csv.reader(f)
        for row in reader:
            CustomRoute.objects.update_or_create(
                auth_route_code=row[0], defaults={'custom_route_code': row[1]})
```

Si se van a agregar try except, agregar las exceptiones específicas que queremos manejar, no hacerlo genérico.

Reply:
Ok